### DISCUSSION TITLE
Any explanation on how to install on raspberry pi

### DISCUSSION THREAD

**Discussion by malkatammuz** [netalertx/NetAlertX#471] [General]
Can anyone provide me with help regarding installing Pi alert on Raspberry Pi.
Thanks in advance

**Comment by jokob-sk** [comment]
Hi there,

The most tested approach would be to setup docker and the install PiAlert as described here: https://github.com/jokob-sk/Pi.Alert/blob/main/dockerfiles/README.md

I have an on-hardware install script which I tested on Ubuntu: https://github.com/jokob-sk/Pi.Alert/blob/main/docs/HW_INSTALL.md

If it's not working on a Raspberry Pi please adjust it and submit an install script for Raspberry Pi if you'd like to do so.

Thanks!
j
